### ISSUE TITLE
Missing totals for additional currency

### ISSUE THREAD

**Issue by alexandrpaliy** [mtotschnig/MyExpenses#970]
MyExpenses version - latest available at Google Store at the moment (3.3.8.1 I believe).

My main currency is UAH (ukrainian hryvnas). For a long time I've also had a couple of accounts in USD, but recently (at least 1 month already) I added some additional accounts in PLN and EUR.
Currently, accounts with PLN/EUR currencies have their balances > 0, but, for some reason, in "Total" section:
1. I see only totals for UAH and USD (but not for PLN and EUR)
2. It looks like "Grand total" is just a sum of only visible totals (so, in my case, "Grand total" is just a sum of UAH and USD (latter auto-converted to UAH, ofc), but PLN and EUR are completely ignored).

If it matters, I use https://openexchangerates.org as an Exchange Rate Provider, and, when creating new non-UAH transaction, I'm able to download actual exchange rates just by using the corresponding button.

Screenshot, which proves the 1st point ("грн" is a cyrillic version of UAH):
![image](https://user-images.githubusercontent.com/5554048/166338563-8fc98803-c9ba-42da-8239-776535346b02.png)
You can see EUR account opened in background, but there is no entry for EUR in Totals.

2nd point can't be proven with sums and balances hidden, so I just ask to believe my words :)

**Comment by mtotschnig** [comment]
A possible explanation for the first point: The aggregate for a currency is only shown for currencies with at least two accounts, because with only one account, it would not add any information.
Another possible explanation: On the account edit screen, there is the option to exclude an account from the calculation of totals. By any chance, might you have activated this option?
If after checking this, the behavior is still not explained, might it be possible for you to create a sample database, and send it to support@myexpenses.mobi ?

**Comment by alexandrpaliy** [comment]
> A possible explanation for the first point: The aggregate for a currency is only shown for currencies with at least two accounts, because with only one account, it would not add any information.

Yes, that's it. I've added another account for EUR, and it appeared in totals. Thanks.

Actually, second point was "the main problem" for me (I wanted to be sure that I see correct Global Total), and I assumed it was caused by the first point.
Checking my accounts' settings just to look at the option you've mentioned

> On the account edit screen, there is the option to exclude an account from the calculation of totals.

... , I actually figured out, what was wrong with the second point.
Turns out, you can/must set an exchange rate in every account settings (it's 1:1 by default). I never watched at that setting, and expected that app will somehow automatically download exchange rates by itself (since I've set up exchange rates provider).
As soon as I've pressed "download" button in PLN/EUR accounts settings, so the app received actual exchange rates - "Global total" became correct (and it doesn't even depend whether "totals" for PLN/EUR are shown or not).


My question is solved now.

As an idea: maybe it makes sense to automatically get the actual exchange rate (instead of 1:1) while creating new account, in case if user has set up Exchange Rate Provider. Or even add an option like "automatically update exchange rate for this account". Just to decrease the probability of confusion similar to mine :)
I'm quite sure I had Exchange Rate Provider successfully set up long before creating new PLN/EUR accounts, and I think I was online at the moment of creation - so I assumed "it will download exchange rate by itself" and didn't even bother looking there.
